src/core/packages/chrome/browser-internal/src/project_navigation/breadcrumbs.tsx

```diff
@@ -33,6 +38,7 @@ export function buildBreadcrumbs({
   activeNodes,
   chromeBreadcrumbs,
   isServerless,
+  echDeploymentName,
 }: {
   projectName?: string;
```

Comment:
maybe we should just unify from here ...
instead of projectName and echDeploymentName just have `kibanaName` or something like that